packages/test/snapshots/src/replayMultipleFiles.ts

```diff
@@ -25,7 +25,6 @@ function getFileLocations(): [string, string] {
     // Relative to this generated js file being executed
     testCollateralPath = nodePath.join(__dirname, "..", testCollateralPath);
     workerPath = nodePath.join(__dirname, "..", workerPath);
-    assert(fs.existsSync(testCollateralPath), `Cannot find test collateral path: ${origTestCollateralPath}`);
```

Comment:
I wonder if we should keep this assert in case we missed the check by the caller?
I know `getFileLocations` is called globally, but that can probably be changed to delay the call until the values are needed, but still only doing the checks once.

Reply:
I think we can do that, yeah, let me give it a shot.

Reply:
Ok, @curtisman that won't work because the caller can't know which path to check for existence (or ask that it gets checked) until after this function runs and computes it. There's a single place where `fileLocation` is used in this file (in the `processContent` function), I could move the `assert` to right before that usage, with a function that keeps track of if it has done the check already, so it only happens once. And keep the exported function that just does an `fs.existsSync(fileLocation)` for other modules to check. I considered `assert`ing in that function but that requires the external module to know to try-catch around the call and ignore that specific kind of error, which feels too leaky. Should I just move the `assert` to where `fileLocation` is used?